src/main/java/rx/internal/util/PlatformDependent.java

```diff
@@ -20,13 +20,19 @@
 
 /**
  * Allow platform dependent logic such as checks for Android.
- * 
+ *
  * Modeled after Netty with some code copy/pasted from: https://github.com/netty/netty/blob/master/common/src/main/java/io/netty/util/internal/PlatformDependent.java
  */
 public final class PlatformDependent {
 
     private static final boolean IS_ANDROID = isAndroid0();
```

Comment:
And delete `isAndroid0()`, there's no reason to have so many mechanisms for checking